pegasus/sites.v3/code.org/public/privacy-apr-2016.md

```diff
@@ -12,7 +12,7 @@ Planning to update privacy policy: Aug 24, 2016
 
 
 ## About Code.org
-Code.org® is a non-profit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented students of color.
+Code.org® is a non-profit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented minorities.
```

Comment:
Previous privacy policies should NOT be changed, they should be maintained for historical record.